### ISSUE TITLE
Checkbox has no icon on IOS

### ISSUE THREAD

**Issue by ForeverYouth** [GeekyAnts/NativeBase#2481]
## I have gone through these following points
- [x]   Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
- [x]   Check latest documentation: https://docs.nativebase.io/
- [x]   Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
- [x]   Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
- []   For discussion purpose make use of NativeBase Slack: http://slack.nativebase.io/
- [x]   For queries related to theme, check [Theme Variables](https://docs.nativebase.io/docs/ThemeVariables.html) from Docs and live NativeBase Theme Editor http://nativebase.io/customizer/

### node, npm, react-native, react and native-base version, expo version if used, xcode version
node: 11.1.0
react-native: 0.57.1
react: 16.5.0
native-base: 2.8.1
expo: 31.0.2
xcode: 10.1
### Expected behaviour
Show CheckBox Icon on iOS
### Actual behaviour on iOS
Don't showed CheckBox Icon
### Any other additional info which would help us debug the issue quicker.

Hi. So, as i written above i had problem with CheckBox Fonts on IOS.

Unexpected behavior
<img width="303" alt="screen shot 2018-12-07 at 21 04 33" src="https://user-images.githubusercontent.com/13434556/49671262-ed0a7700-fa6f-11e8-856b-a928fe1785e8.png">
Console writes next issue:
<img width="1250" alt="screen shot 2018-12-07 at 21 12 39" src="https://user-images.githubusercontent.com/13434556/49671321-25aa5080-fa70-11e8-8692-e8a7ca4f97d8.png">

**Comment by schie** [comment]
I am having the same issue.

Any update on this?

**Comment by suvenduchhatoi** [comment]
@ForeverYouth Checkbox uses react-native-vector-icons inside, which seems to have not loaded for some reason. Please check if react-native-vector-icons works in your project properly. If not please link it and that should fix this issue.

**Comment by ForeverYouth** [comment]
@suvenduchhatoi . Yes i know. And probably i've found problem. npm installs wrong version of react-native-vector-icons. 
In git repository of native-base library in package.json react-native-vector-icons version are 6.0.2 but npm for some reason install wrong version of react-native-vector-icons (4.6.0) in node_modules/native-base/node_modules.
How can solve this problem?

**Comment by suvenduchhatoi** [comment]
@ForeverYouth 
Try uninstalling native-base by running commands:
```react-native unlink native-base
npm uninstall native-base```

You could remove your node_modules/ folder and then reinstall the dependencies from package.json.
```rm -rf node_modules/
npm install```

Then re-install native-base
```npm install native-base --save
react-native link```
Hopefully, it should work...

**Comment by ForeverYouth** [comment]
@suvenduchhatoi unfortunately it didn't help. Because i use CRNA. Depend on tutorial i don't need use react-native link. Even i do it - nothing changes - same result :(

**Comment by bagen1** [comment]
@ForeverYouth how did you style the checkmark circle to be beside the text?

**Comment by SupriyaKalghatgi** [comment]
@ForeverYouth @suvenduchhatoi This is fixed on branch `master`
Refer existing / closed issues before initiating one

**Comment by SupriyaKalghatgi** [comment]
Released 2.9.0
